### DISCUSSION TITLE
Much more found IsoformSwitches with different GTF file

### DISCUSSION THREAD

**Discussion by Pinolinoo** [kvittingseerup/IsoformSwitchAnalyzeR#130] [General]
I have another question: From the resulting Isoformswitcher analysis with the mixed merged gtf file I get now much more isoform switches in the range of up to 2700 thousand per comparison whereas before when I used only the short read assambled gtf I had around 300-600 per condition. Why is that the case?

**Comment by kvittingseerup** [comment]
That is a difficult question to answer. What is the overlap between the different assembly approaches?

Also this is quantified using both short and long reads right?

**Reply by Pinolinoo** [reply]
For the quantification I only used the short reads with the different GTF files!

**Reply by kvittingseerup** [reply]
Guess the combined file gives you much more power. If you want to know more you would probably have to dig into the similarities and differences between the two references :)
